runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5878,8 +5879,19 @@ class INTERPRETER_CLASS
 		case J9VM_CONTINUATION_RETURN_FROM_JIT_MONITOR_ENTER: {
 			rc = tryEnterBlockingMonitor(REGISTER_ARGS, syncObject, J9VM_CONTINUATION_RETURN_FROM_JIT_MONITOR_ENTER);
 			if ((NULL != _currentThread->currentContinuation) && (EXECUTE_BYTECODE == rc)) {
-				void *returnAddress = restoreJITResolveFrame(REGISTER_ARGS);
-				VMStructHasBeenUpdated(REGISTER_ARGS);
+				J9SFJITResolveFrame *resolveFrame = (J9SFJITResolveFrame*)_currentThread->sp;
+				void *returnAddress = resolveFrame->returnAddress;
```

Comment:
It might be clearer to initialize this to NULL here, and set it inside the two cases (both cases set the variable to the same value anyway).